src/Visitor.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace HexletPsrLinter;
+
+use PhpParser\Node;
+use PhpParser\NodeVisitorAbstract;
+
+class Visitor extends NodeVisitorAbstract
+{
+    private $errorFunctions = [];
+
+    public function leaveNode(Node $node)
+    {
+        $nodeClass = get_class($node);
+        if ($nodeClass == "PhpParser\\Node\\Stmt\\ClassMethod" && checkFunctionName($node->name) != true) {
+            $error_string = "Method name is not in camel caps format";
```

Comment:
Так переменные можно же задавать в формате underscore.
http://www.php-fig.org/psr/psr-1/#4-2-properties

On Fri, Aug 5, 2016 at 10:14 AM, Roman Lakhtadyr <notifications@github.com>
wrote:

> In src/Visitor.php
> <https://github.com/maximtop/hexlet-psr-linter/pull/1#discussion_r73651060>
> :
>
> > +<?php
> > +
> > +namespace HexletPsrLinter;
> > +
> > +use PhpParser\Node;
> > +use PhpParser\NodeVisitorAbstract;
> > +
> > +class Visitor extends NodeVisitorAbstract
> > +{
> > +    private $errorFunctions = [];
> > +
> > +    public function leaveNode(Node $node)
> > +    {
> > +        $nodeClass = get_class($node);
> > +        if ($nodeClass == "PhpParser\\Node\\Stmt\\ClassMethod" && checkFunctionName($node->name) != true) {
> > +            $error_string = "Method name is not in camel caps format";
>
> $error_string - пишете линтер, а сами правилам чего не следуете? :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/maximtop/hexlet-psr-linter/pull/1/files/de77fd2c026aec74dad587f6eb61c5e151b5f7c4#r73651060>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AQePwXMxOy8IPm-QTdlxN-i0SmKAyqLTks5qcuLngaJpZM4JdWB2>
> .
>



-- 
Sincerely yours,
Maxim Nikolaevich
mob. 8-926-934-08-25
email: maximtop@gmail.com